guvnor-ala/guvnor-ala-services-rest/src/test/java/org/guvnor/ala/services/rest/tests/RestPipelineImplTest.java

```diff
@@ -185,6 +188,11 @@ public static Archive createDeployment() throws Exception {
         return deployment;
     }
 
+    @BeforeClass
+    public static void beforeClass() {
+        Assume.assumeFalse(System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("windows"));
```

Comment:
Considering that this `beforeClass()` is repeated in 3 different tests, wouldn't it make sense to create a new parent class for these tests and put the method there? Let's not duplicate code.

Reply:
I don't think this would be good, because the classes are in different packages that are probably unrelated. However I am not against having a util class with a method that checks os type. E.g. like isWindows(). This method can then be used in the assumes here. Later, we can search for this method references so we will always know where we have put these assumes.

Reply:
Maybe this [1] can be also used when commons-lang is in the dependencies list. 

[1] http://grepcode.com/file/repo1.maven.org/maven2/commons-lang/commons-lang/2.6/org/apache/commons/lang/SystemUtils.java#SystemUtils.0IS_OS_WINDOWS